### PR TITLE
36677 - 36678 - Finish create, index, destroy send request book

### PR DESCRIPTION
**Pull request by vinhnd2010**

### CODE REVIEW

File: app/views/requests/_request.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<li>
+  <%= link_to request.book_name, "" %>
+  <%= request.author %>
+  <%= request.pulisher %>
+  | <%= link_to "cacel", request, method: :delete,
```
Comment (ThuBM) [inline app/views/requests/_request.html.erb]: cacel????

File: app/controllers/requests_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    if @request.save
+      flash[:success] = t "flash.request.success"
+      redirect_to root_url
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    Request.find(params[:id]).destroy
```
Comment (ThuBM) [inline app/controllers/requests_controller.rb]: check if fails

**Comment by ThuBM** [comment]
ok

File: app/controllers/requests_controller.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+class RequestsController < ApplicationController
+  before_action :user_signed_in?
```
Comment (trinhducduy) [inline app/controllers/requests_controller.rb]: should be authenticate_user! instead of user_signed_in?

**Pull request merged**